docs/review-merge-process.md

```diff
@@ -0,0 +1,23 @@
+# Mappings Review / Merge Process
+
+This document details the review and merge process for Parchment mappings.
+
+### Requirements to merge:
+
+- If we have any automatic tests, they all must pass.
+  - Example: A PR may not be merged if the Contributor License Agreement (CLA) is not signed.
+- A PR requires two approving reviews
+  - One approving review is required to be from a Parchment team member.
+- We ask that people don’t review things that they don’t actually have the capability to effectively review (knowledge wise).
+- All external reviewers are welcome, but not required.
+
+On Resolving conflicting opinions:
+
+- Any names involving substantial conflicts should be available for public input for some length of time (i.e. a discussion time) before merging. After such time, the PR may be merged if there is:
+  1. A simple majority of Parchment team reviewers
+  2. In the event of a tie, A simple majority of all reviewers
+- Submissions should be substantial. (Single line typo fix PRs are discouraged)
+- Omnibus PRs may be made ocassionally to accumulate lots of smaller fixes.
+- Contributions must require a github authentication, to link contributors back to individual contributions.
+- Parchment Team / Reviewers can be free to merge and/or review at any time, based on time.
```

Comment:
Should we have a small delay between the last required approval reivew and the actual merge? This would give both the reviewers team and external reviewers a window of time to confirm that nothing is majorly out of place.

Reply:
Unsure. A valid question to bring up. I don't terribly mind either way.

Reply:
Discussion on the Discord has led to a recommendataion of a minimum of 48 hours between the last approving review required for merging, and the actual merging itself.

However, I still firmly believe that an option of a minimum of 24 hours be inlcuded, for small uncontestable submissions (such as typo fixes, or minor grammatical changes).